# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsContentPolicyUtils.h
# Commit: 1700d5b79273
# Full Hash: 1700d5b79273a6f34d67b7d16470973f0cb65793
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-12-21 21:21:23
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Implement csp for Module Workers; r=evilpie,asuth,rpl,ckerschb
#   
#   Depends on D155691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156102
# ==============================================================================

diff -r 31888ffde37a -r 1700d5b79273 dom/base/nsContentPolicyUtils.h
--- a/dom/base/nsContentPolicyUtils.h	Tue Dec 20 20:56:15 2022 +0000
+++ b/dom/base/nsContentPolicyUtils.h	Tue Dec 20 20:56:15 2022 +0000
@@ -144,8 +144,10 @@
     CASE_RETURN(TYPE_INTERNAL_FRAME_MESSAGEMANAGER_SCRIPT);
     CASE_RETURN(TYPE_INTERNAL_FETCH_PRELOAD);
     CASE_RETURN(TYPE_UA_FONT);
+    CASE_RETURN(TYPE_INTERNAL_WORKER_STATIC_MODULE);
     CASE_RETURN(TYPE_PROXIED_WEBRTC_MEDIA);
     CASE_RETURN(TYPE_WEB_IDENTITY);
+    CASE_RETURN(TYPE_END);
     case nsIContentPolicy::TYPE_INVALID:
       break;
       // Do not add default: so that compilers can catch the missing case.